stdlib/public/TensorFlow/Ops.swift

```diff
@@ -1259,16 +1259,14 @@ public extension Tensor where Scalar : Numeric {
     where Scalar : TensorFlowFloatingPoint
   )
   func sum() -> Tensor {
-    let axes = Tensor<Int32>(rangeFrom: 0, to: rank, stride: 1)
-    return Raw.sum(self, reductionIndices: axes)
+    return Raw.sum(flattened(), reductionIndices: [0])
```

Comment:
We need a shape-squeezed result (i.e. a scalar) from these whole-tensor reduction operators. Could you add some tests?